pkg/config/user_config.go

```diff
@@ -10,50 +10,50 @@ type UserConfig struct {
 	// Config relating to git
 	Git GitConfig `yaml:"git"`
 	// Periodic update checks
-	Update UpdateConfig `yaml:"update"`
+	Update UpdateConfig `yaml:"update" jsonschema:"additionalProperties=false"`
 	// Background refreshes
-	Refresher RefresherConfig `yaml:"refresher"`
+	Refresher RefresherConfig `yaml:"refresher" jsonschema:"additionalProperties=false"`
 	// If true, show a confirmation popup before quitting Lazygit
 	ConfirmOnQuit bool `yaml:"confirmOnQuit"`
 	// If true, exit Lazygit when the user presses escape in a context where there is nothing to cancel/close
 	QuitOnTopLevelReturn bool `yaml:"quitOnTopLevelReturn"`
 	// Keybindings
-	Keybinding KeybindingConfig `yaml:"keybinding"`
+	Keybinding KeybindingConfig `yaml:"keybinding" jsonschema:"example=<enter>,example=<c-b>,example=g"`
```

Comment:
These examples don't look right. This is the container for all keybindings, not a single key binding, so it doesn't make sense to have this here. It also didn't make it into the schema, probably because you can't have examples on an object.

You'd really want to have these on every single keybinding, but I'm not sure how to achieve that, or how hard we should try.

Reply:
Removed in: dfddb863371a8f300718a6d1f575246bf78be0a8

could use ``

Reply:
Couldn't you do it in the same way as the Pager examples? Have `type KeybindingType string`, use that for all bindings, and then use `JSONSchemaExtend` to attach the examples? I'm not sure it's important enough though.

Reply:
Yeah, that would also work. We would just have to find all the places these keybinds go and remember to cast them to strings. Is that a trade off we would be willing to take?

Reply:
Finding them is not the issue, the compiler will tell us. Whether it's worth it, I'm not sure; maybe not.